.github/workflows/tests.yml

```
@@ -0,0 +1,31 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r docs/requirements.txt
```

pytest is documented as a development dependency in `pyproject.toml`:

https://github.com/KnowledgeEdgeAI/PETs_for_Public_Health_Challenge/blob/210239cdc25bd3299f8765bb5754ab52650bba31/pyproject.toml#L39-L42

so you can install the package with the requirement dependencies by running:

```suggestion
        python -m pip install .[dev]
```

This has the benefit that you don't mix the package dependencies with the dependencies to build the documentation.